dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04210CreateDefaultLanguageVariable.java

```diff
@@ -0,0 +1,183 @@
+            // Insert into Inode table the reference to the field
+            dc.setSQL(INSERT_INODE_QUERY);
+            dc.addParam(LANGUAGE_VARIABLE_FIELD_INODES[i]);
+            dc.addParam(StringPool.BLANK);
+            dc.addParam(new Date());
+            dc.addParam("field");
+            dc.loadResult();
+            // Insert into Field table
+            dc.setSQL(INSERT_FIELD_QUERY);
+            dc.addParam(LANGUAGE_VARIABLE_FIELD_INODES[i]);
+            dc.addParam(field.get("structureInode"));
+            dc.addParam(field.get("fieldName"));
+            dc.addParam(field.get("fieldType"));
+            dc.addParam(field.get("fieldRelationType"));
+            dc.addParam(field.get("fieldContentlet"));
+            dc.addParam(field.get("isRequired"));
+            dc.addParam(field.get("isIndexed"));
+            dc.addParam(field.get("isListed"));
+            dc.addParam(field.get("velocityVarName"));
+            dc.addParam(field.get("sortOrder"));
+            dc.addParam(field.get("values"));
+            dc.addParam(field.get("regexCheck"));
+            dc.addParam(field.get("hint"));
+            dc.addParam(field.get("defaultValue"));
+            dc.addParam(field.get("isFixed"));
+            dc.addParam(field.get("isReadOnly"));
+            dc.addParam(field.get("isSearchable"));
+            dc.addParam(field.get("isUnique"));
+            dc.addParam(new Date());
+            dc.loadResult();
+            i++;
+        }
+    }
+
+    /**
+     * Creates each field that is required by the Language Variable Content Type and returns a list
+     * with them.
+     * 
+     * @return The List of Language Variable fields.
+     */
+    @SuppressWarnings("unchecked")
+    private List<Map<String, Object>> getLanguageVariableFields() {
+        final boolean isRequired = Boolean.TRUE;
+        final boolean isListed = Boolean.TRUE;
+        final boolean isIndexed = Boolean.TRUE;
+        final boolean isFixed = Boolean.TRUE;
+        final boolean isReadOnly = Boolean.TRUE;
+        final boolean isSearchable = Boolean.TRUE;
+        final boolean isUnique = Boolean.TRUE;
+        Map<String, Object> keyField = setFieldElements(KeyValueContentType.KEY_VALUE_KEY_FIELD_NAME,
+                        LegacyFieldTypes.TEXT.implClass().getName(), StringPool.BLANK, "text1", isRequired, isListed, isIndexed,
+                        KeyValueContentType.KEY_VALUE_KEY_FIELD_VAR, 1, StringPool.BLANK, StringPool.BLANK, StringPool.BLANK,
+                        StringPool.BLANK, isFixed, !isReadOnly, isSearchable, !isUnique);
+        Map<String, Object> valuField = setFieldElements(KeyValueContentType.KEY_VALUE_VALUE_FIELD_NAME,
+                        LegacyFieldTypes.TEXT_AREA.implClass().getName(), StringPool.BLANK, "text_area1", isRequired, isListed,
+                        !isIndexed, KeyValueContentType.KEY_VALUE_VALUE_FIELD_VAR, 2, StringPool.BLANK, StringPool.BLANK,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 2 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS109)